addons/account/static/src/components/open_move_line_move_widget/open_move_line_move_widget.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import {Many2OneField} from "@web/views/fields/many2one/many2one_field";
+
+class OpenMoveLineMoveWidget extends Many2OneField {
```

Comment:
Would it not be better to modify or extend `OpenMoveWidget`?

There's some options:
1. pass the id to the openMove function (and change/extend the xml to use this argument)
2. create a form version with an overridden openMove function (in the same file)

But I guess you want the features of Many2oneField in edit mode (in that case, ignore this)

Reply:
Maybe OpenMoveWidget should be changed to use the `move_id` field instead of `move_name`
that way you can reuse the widget for both fields.

Reply:
I wanted to reuse the widget but:
I need the Many2oneField features, so it can't be shared.
Also, we want one widget to open upon click, and for the other, it's the m2o action that changes. So I don't think they can be merged, unfortunately. The action on click doesn't work well with m2o as it conflicts with the click to open the list.

Reply:
I would call the component `LineOpenMoveWidget` and the widget `line_open_move_widget` for better greps

Reply:
Ok, I will change the names :)
Benj thought of it and said it's still what we want, in his opinion most of the time the move line form view is only a step in order to open the move itself. You may challenge this with him if you think this is not needed